tests/functional/adapter/test_python_model.py

```diff
@@ -20,10 +20,26 @@ def project_config_update(self):
 
 models__simple_python_model = """
 import pandas
+import torch
+import spacy
 
 def model(dbt, spark):
     dbt.config(
         materialized='table',
+        submission_method='job_cluster',
+        job_cluster_config={
+            "spark_version": "7.3.x-scala2.12",
+            "node_type_id": "i3.xlarge",
+            "num_workers": 0,
```

Comment:
@dbeatty10  this is how to get it as single node per doc [here](https://docs.databricks.com/dev-tools/api/latest/clusters.html#examples)

Reply:
Nice! 💯